### PR TITLE
3 1 0 upgrade

### PR DESCRIPTION
**Pull request by DavidGoldwasser**
All tests on dashboard are passing on my local Mac with latest 3.1 alpha installer.

### CODE REVIEW

**Comment by DavidGoldwasser** [comment]
3 of the generic_QAQC tests are failing on CI because of this. 

` Error prevented QAQC check from running (undefined method `model_get_construction_set' for #<ASHRAE9012013:0x0000560361a84688>)`

They run on my mac. I can test on Window, but @tijcolem is it possible CI is using a different standards gem than is in the last installer?

**Comment by DavidGoldwasser** [comment]
@nllong should be all ready, I updated to use final extension and standards gem for OpenStudio 3.1.0

File: lib/measures/openstudio_results/tests/os_results.osw [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (nllong) [inline lib/measures/openstudio_results/tests/os_results.osw:1]: should this file be in here? seem likes it is a result.

File: openstudio-common-measures.gemspec [hunk 2]
```diff
@@ -29,8 +29,8 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '~> 2.5.0'
 
   spec.add_dependency 'bundler', '~> 2.1'
-  spec.add_dependency 'openstudio-extension', '~> 0.2.3'
-  spec.add_dependency 'openstudio-standards', '~> 0.2.11'
+  spec.add_dependency 'openstudio-extension', '~> 0.3.1'
```
Comment (nllong) [inline openstudio-common-measures.gemspec:32]: 👍

**Review by nllong — CHANGES_REQUESTED** [review]

File: lib/measures/openstudio_results/tests/os_results.osw (hunk 1 above)
Comment (DavidGoldwasser) [reply lib/measures/openstudio_results/tests/os_results.osw:1]: @nllong It should stay it is an input for a test. https://github.com/NREL/openstudio-common-measures-gem/blob/3_1_0_upgrade/lib/measures/openstudio_results/tests/OpenStudioResults_Test.rb#L85

File: openstudio-common-measures.gemspec (hunk 2 above)
Comment (DavidGoldwasser) [reply openstudio-common-measures.gemspec:32]: These needed to be updated, but long term I'd like to not have these here, it just creates headaches, for example testing CI with multi versions of OS but getting the right gems.

Comment (DavidGoldwasser) [reply openstudio-common-measures.gemspec:32]: Bundle install also breaks on everything using measure gems unless they are all in sync. Maybe it is needed for developer environment, but should need to install OpenStudio standards to do bundle install of measure gems.

**Review by DavidGoldwasser — COMMENTED** [review]
@nllong I can submit comments here, but Approve is commented out for me.

**Pull request merged**